content/vendor/guides/ha-guide.md

```diff
@@ -0,0 +1,411 @@
+
+```shell
+curl -sSL https://k8s.kurl.sh/appdirect-unstable | sudo bash -s ha
+```
+
+When you include the '-s ha' option, you will be promoted right away for the load balancer address. As shown in the figure below, we are providing the 'internal' IP address. This address is used by the Kubernetes services in the nodes to talk to each other. Using a public IP addresses addds extra layers that are not needed. 
+
+This process will take several minutes as it will install Kubernetes as well as the Application Administration toolset. Once it is finished, you should see something similar as shown below:
+
+![Output](/images/guides/kots/ha-cluster-install-output.png)
+
+Highlighted in red in the screenshot above are first the Kots Admin URL and password to login. Also highlighted are the commands to run on other nodes to join the cluster. In this exercise we are not going to run these commands yet and instead continue deploying the application. The commands to join the cluster are also available in the Kots Admin UI, which we will cover later in this guide.
+
+Next, we are going to log in to Kots Admin to complete the install and deploy the application. Since this guide assumes you have a level of familiarity with Kots, we are not going to go into detail on every step to get the application deployed.
+
+![Log-In](/images/guides/kots/ha-cluster-log-in.png)
+
+Log in to the Aplication Administration Console using the address and password from the install. Once you have logged in using the provided password, you will need to provide a license. The license is generated in the Vendor Portal and this guide assumes you already know how to do this. There are no options that need to be enabled or turned on in order to support an HA cluster, so if you already have a devlopment license, it should work for this exercise.
+
+![Add-License](/images/guides/kots/ha-cluster-add-license.png)
+
+Once you upload your license it may take a few minutes to load and continue with the deployment.
+
+
+![License-Load](/images/guides/kots/ha-cluster-license-load.png =250x250)
+
+Once the license is loaded, the next window will depend on the application being deployed as this window renders whatever has been codified in the application's [Config.yaml](https://kots.io/reference/v1beta1/config/).
+
+After the configuration window, you should see the preflight checks and eventually land on a window similar to what is shown below
+
+![KotsAdmin](/images/guides/kots/ha-cluster-app-dployed.png)
+
+
+### Verify the Deployment
+
+To see everything that has been deployed, ssh into one of the nodes and run:
+
+```shell
+kubectl get pods --all-namespaces -o wide
+```
+
+
+The output will show you all the pods running on all namespaces and on which node. Since we only have a one node cluster, all pods should be running on node 1. Check the status of the pods to make sure there are no issues.
```

Comment:
This first sentence sounds a little awkward to me -- particularly the last bit about `on which node`

Reply:
I broke it down into two sentences:

The output will show you all the pods running on all namespaces. The output also shows you the node that each pod is currently running on, which will come in handy during this exercise.